test/Tool/power2.opt

```diff
@@ -0,0 +1,55 @@
+; REQUIRES: solver
```

Comment:
this one seems a bit brute-force, why not just assert that the ctpop == 1?

Reply:
I wrote this test case to make sure we have all power of two numbers (including INT_MIN) this time. ctpop anyways returned 1 when I didn't include 1 << (bitwidth-1) in my KLEE query earlier. I can add ctpop==1 test case too. Should I get rid of this long brute force test?

Reply:
also this test passes with current Souper. so it doesn't test your new code.

Reply:
Yes, you're right. I have checked the leading zeros on the power of two now. This confirms the bug is fixed as the invalid and LGTM version of same test case just differ on no leading zeros for INT_MIN case.